### PR TITLE
Block Indicators - Generic v2/v3 block non-malicious hashes by default

### PR DESCRIPTION
**Pull request by TalNos**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-2781

## Description
- Now by default, inputs are configured to block only malicious files within the following playbooks:
1- Block_Indicators_-_Generic_v2
2- Block_Indicators_-_Generic_v3

- Added missing playbook input in Block URL - Generic v2 playbook.

## Screenshots
Block_Indicators_-_Generic_v2 playbook:
Block_Indicators_-_Generic_v2

Block_Indicators_-_Generic_v3 playbook:
Block_Indicators_-_Generic_v3

Block_URL_-_Generic_v2 playbook:
Block_URL_-_Generic_v2

### CODE REVIEW

File: Packs/CommonPlaybooks/Playbooks/playbook-Block_Indicators_-_Generic_v2.yml [hunk 1]
```diff
@@ -603,6 +759,5 @@ outputs:
 - contextPath: CheckpointFWRule.Hits.Value
   description: Value of rule hits.
 tests:
-- No Test
+- No tests (auto formatted)
 fromversion: 5.0.0
-toversion: 5.4.9
```
Comment (idovandijk) [inline Packs/CommonPlaybooks/Playbooks/playbook-Block_Indicators_-_Generic_v2.yml:608]: The `toversion` should stay the same. Congratulations on finding a new SDK bug!
Please open a Jira ticket about this. If a playbook that already exists in content has a `toversion`, the SDK should keep it.

File: Packs/CommonPlaybooks/ReleaseNotes/2_3_7.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+#### Playbooks
+##### Block Indicators - Generic v2
```
Comment (idovandijk) [inline Packs/CommonPlaybooks/ReleaseNotes/2_3_7.md:3]: Add the v3 playbook as well

File: Packs/CommonPlaybooks/Playbooks/playbook-Block_Indicators_-_Generic_v2_README.md [hunk 3]
```diff
@@ -87,4 +92,4 @@ This playbook does not use any commands.
 
 ## Playbook Image
 ---
-![Block Indicators - Generic v2](https://raw.githubusercontent.com/demisto/content/1077ac4f6a51a75ab0aa149785095aad3a8757fa/Packs/CommonPlaybooks/doc_files/Block_Indicators_-_Generic_v2.png)
+![Block Indicators - Generic v2](../doc_files/Block_Indicators_-_Generic_v2.png)
```
Comment (idovandijk) [inline Packs/CommonPlaybooks/Playbooks/playbook-Block_Indicators_-_Generic_v2_README.md:95]: nice job

File: Packs/CommonPlaybooks/Playbooks/playbook-Block_Indicators_-_Generic_v2_README.md [hunk 4]
```diff
@@ -1,20 +1,23 @@
-This playbook blocks malicious indicators through all integrations that are enabled, using the following sub-playbooks:
```
Comment (idovandijk) [inline Packs/CommonPlaybooks/Playbooks/playbook-Block_Indicators_-_Generic_v2_README.md:1]: Good job updating this, let's also add the README of the v3

**Review by idovandijk — CHANGES_REQUESTED** [review]
Excellent job Tal!
Please see my comments. Also, because this playbook is a very popular one, and changes are very sensitive, let's use the opportunity to update the test playbook.
You can take the test playbook YML from the CommonPlaybooks pack under the TestPlaybooks folder. Upload it to your server and create an incident with malicious MD5 / SHA256, then run the playbook on that incident, and then verify that the the malicious hashes are passed to the playbook inputs to be blocked.

File: Packs/CommonPlaybooks/Playbooks/playbook-Block_Indicators_-_Generic_v3.yml [hunk 5]
```diff
@@ -616,12 +754,6 @@ inputs:
...
   playbookInputQuery:
-- key: URLListName
-  value:
-    simple: Demisto Remediation - URL EDL
-  required: false
-  description: URL list from the instance context with which to override the remote file.
-  playbookInputQuery:
 - key: CustomURLCategory
   value:
     simple: Demisto Remediation - Malicious URLs
```
Comment (idovandijk) [inline Packs/CommonPlaybooks/Playbooks/playbook-Block_Indicators_-_Generic_v3.yml:759]: `Block URL - Generic v2` is using "XSOAR Remediation - Malicious URLs". Please change the input for `Block Indicators - Generic v3` to be "XSOAR Remediation - Malicious URLs" and then in `Block URL - Generic v2` use the input. This way we fix it to use the input while preserving backward compatibility

**Review by idovandijk — CHANGES_REQUESTED** [review]
small change

**Review by idovandijk — APPROVED** [review]
Looks good, well done

**Comment by ShirleyDenkberg** [comment]
@idovandijk Doc review (RN only) completed.

**Pull request merged**